tests/system/python/conftest.py

```diff
@@ -188,6 +201,45 @@ def _read_data_from_pi(host, admin, password, pi_database, asset, sensor):
     return _read_data_from_pi
 
 
+@pytest.fixture
+def add_filter():
+    def _add_filter(filter_plugin, filter_plugin_branch, filter_name, filter_config, foglamp_url, filter_user_svc_task):
+        """
+
+        :param filter_plugin: filter plugin `foglamp-filter-?`
+        :param filter_plugin_branch:
+        :param filter_name: name of the filter with which it will be added to pipeline
+        :param filter_config:
+        :param foglamp_url:
+        :param filter_user_svc_task: south service or north task instance name
+        """
+
+        try:
+            subprocess.run(["$FOGLAMP_ROOT/tests/system/python/scripts/install_c_plugin {} filter {}"
+                               .format(filter_plugin_branch, filter_plugin)], shell=True, check=True)
+        except subprocess.CalledProcessError:
+            assert False, "{} plugin installation failed".format(filter_plugin)
+
+        data = { "name": "{}".format(filter_name), "plugin": "{}".format(filter_plugin), "filter_config": filter_config}
+        conn = http.client.HTTPConnection(foglamp_url)
+
+        conn.request("POST", '/foglamp/filter', json.dumps(data))
+        r = conn.getresponse()
+        assert 200 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        assert filter_name == jdoc["filter"]
+
+        uri = "{}/pipeline?allow_duplicates=true&append_filter=true".format(quote(filter_user_svc_task))
+        filters_in_pipeline = [filter_name]
+        conn.request("PUT", '/foglamp/filter/' + uri, json.dumps({"pipeline": filters_in_pipeline}))
+        r = conn.getresponse()
+        assert 200 == r.status
+        res = r.read().decode()
+        print(res)
```

Comment:
Remove print
```
 exp = "Filter pipeline {{'pipeline': ['{}']}} updated successfully".format(filter_name)
 jdoc = json.loads(res)
 assert exp == jdoc["result"]
```

Reply:
just FYI print is okay, in case of fixtures when we want to see console output with -s flag. but its usage should be rare and justfied